source/installing.rst

```diff
@@ -14,15 +14,12 @@ Like many other |percona| products, |pdp| is installed by using the
 
 .. important::
 
-   Before you attempt to install |pdp|, reinstall |percona-release| to get its latest
-   version. 
+   Before you attempt to install |pdp|, reinstall |percona-release| to
```

Comment:
maybe we should say `update percona-release to the latest version`?

Reply:
What steps should be done in order to update already existent percona-release package:
1 enable original repo
2 run `yum update percona-release` or `apt-get install percona-release`
And in this case user doesn't neet to download anything manually and find what version of percona-release is the latest one